drivers/dma/bcm2835-dma.c

```diff
@@ -171,6 +171,17 @@ struct bcm2835_desc {
 #define WAIT_RESP(x) ((x & BCM2835_DMA_NO_WAIT_RESP) ? \
 		      0 : BCM2835_DMA_WAIT_RESP)
 
+/* A fake bit to request that the driver requires wide reads */
+#define BCM2835_DMA_WIDE_SOURCE BIT(28)
+#define WIDE_SOURCE(x) ((x & BCM2835_DMA_WIDE_SOURCE) ? \
```

Comment:
It might be wise to put some parentheses around the `x` here (and WAIT_RESP and WIDE_DEST), just in case.